decidim-elections/app/views/decidim/votings/admin/votings/new.html.erb

```diff
@@ -1,5 +1,4 @@
 <% add_decidim_page_title(t("votings.new.title", scope: "decidim.votings.admin")) %>
-
```

Comment:
I see that you don't like having the new line here. I wanted to actually create a new `erblint` rule for this, with just the opposite.

In my opinion, there should be an empty line between the helper methods that modify the header or metadata, such as content_for or append_stylesheet_pack_tag.

No need to change anything for now, just wanted to mention that 😄